### PR TITLE
Server 12.12 rel notes and config additions

### PR DESCRIPTION
**Pull request by davidwrede**
- Added server 12.12 release notes
- Added new config settings to chef server rb config doc
- Filled in some previously missing config settings
- Updated note about `chef-server-ctl backup` command.

### CODE REVIEW

File: chef_master/source/config_rb_server_optional_settings.rst [hunk 1]
```diff
@@ -61,6 +61,9 @@ This configuration file has the following general settings:
 ``flavor``
```
Comment (stevendanna) [inline chef_master/source/config_rb_server_optional_settings.rst:61]: Should we add the gc_log configuration to this file as well?

**Review by stevendanna — COMMENTED** [review]
Thanks for doing this!  They look good overall, left one minor comment.

File: chef_master/source/config_rb_server_optional_settings.rst (hunk 1 above)
Comment (davidwrede) [reply chef_master/source/config_rb_server_optional_settings.rst:61]: Yup. PR will be updated shortly.

File: chef_master/source/config_rb_server_optional_settings.rst [hunk 2]
```diff
@@ -1150,6 +1163,9 @@ This configuration file has the following settings for ``opscode-solr4``:
 
       /var/log/opscode/opscode-solr4
 
+``opscode-solr4['log_gc']``
```
Comment (stevendanna) [inline chef_master/source/config_rb_server_optional_settings.rst]: This should be `_` rather than `-`

**Review by stevendanna — APPROVED** [review]

**Comment by stevendanna** [comment]
Chef 12.12 is released. Merge whenever.

**Comment by davidwrede** [comment]
@stevendanna Apologies for the delay. Merging now.

**Pull request merged**